pylearn2/datasets/svhn.py

```diff
@@ -29,14 +30,28 @@ class SVHN(dense_design_matrix.DenseDesignMatrixPyTables):
 
     Parameters
     ----------
-    which_set : WRITEME
-    path : WRITEME
-    center : WRITEME
-    scale : WRITEME
-    start : WRITEME
-    stop : WRITEME
-    axes : WRITEME
-    preprocessor : WRITEME
+    which_set : str
+                Subset name.
+    path : str, optional
+           Path to pytable h5 files in case you want to use
+           preprocessed/modified data.
+    center : bool, optional
+             Center data around zero.
```

Comment:
How is this done? On a per-image or per batch basis? I'm assuming you're not actually subtracting the mean of the hundreds of thousands of images...